examples/bert/bert_auto_opt_gpu.json

```diff
@@ -16,6 +16,22 @@
             }
         }
     },
+    "systems": {
+        "local_system": {
+            "type": "LocalSystem",
+            "config": {
+                "accelerators": [
+                    {
+                        "device": "gpu",
```

Comment:
Should we have some examples where the device is not provided if it is clear from the ep? Isn't that the motivation for the feature implemented in https://github.com/microsoft/Olive/pull/428 @trajepl?

Reply:
Just remove the device attribute should be fine.

Reply:
see https://github.com/microsoft/Olive/blob/359fe22e3f8679de61ff3a56e0b9958c11479968/test/unit_test/hardware/test_accelerator.py#L76

Reply:
yes. I was asking if we should have some examples that do that. Or keep all of the examples with complete configs.